docs/data/material/pages.ts

```diff
@@ -158,64 +162,81 @@ const pages: MuiPage[] = [
     children: [
+      { pathname: '/material-ui/customization/how-to-customize' },
+      { pathname: '/material-ui/customization/dark-mode' },
+      { pathname: '/material-ui/customization/color' },
+      { pathname: '/material-ui/customization/right-to-left', title: 'Right-to-left' },
+      { pathname: '/material-ui/customization/shadow-dom', title: 'Shadow DOM' },
       {
         pathname: '/material-ui/customization/theme',
         subheader: '/material-ui/customization/theme',
         children: [
-          { pathname: '/material-ui/customization/theming' },
+          { pathname: '/material-ui/customization/default-theme', title: 'Default theme viewer' },
+          { pathname: '/material-ui/customization/theming', title: 'Customizing the theme' },
+          {
+            pathname: '/material-ui/customization/creating-themed-components',
+            title: 'Creating themed components',
+          },
+          { pathname: '/material-ui/customization/theme-components', title: 'Components' },
+        ],
+      },
+      {
+        pathname: '/material-ui/customization/tokens',
+        subheader: 'tokens',
+        children: [
           { pathname: '/material-ui/customization/palette' },
-          { pathname: '/material-ui/customization/dark-mode' },
           { pathname: '/material-ui/customization/typography' },
           { pathname: '/material-ui/customization/spacing' },
           { pathname: '/material-ui/customization/breakpoints' },
           { pathname: '/material-ui/customization/density' },
           { pathname: '/material-ui/customization/z-index', title: 'z-index' },
           { pathname: '/material-ui/customization/transitions' },
-          { pathname: '/material-ui/customization/theme-components', title: 'Components' },
-          { pathname: '/material-ui/customization/default-theme', title: 'Default theme viewer' },
         ],
       },
-      { pathname: '/material-ui/customization/how-to-customize' },
-      { pathname: '/material-ui/customization/color' },
     ],
   },
   {
     pathname: '/material-ui/guides',
     title: 'How-to guides',
     children: [
-      { pathname: '/material-ui/guides/api', title: 'API design approach' },
       {
-        pathname: '/material-ui/guides/creating-themed-components',
-        title: 'Creating themed components',
+        pathname: '/material-ui/guides/material-3-components',
+        title: 'Material 3 components',
+        newFeature: true,
       },
-      { pathname: '/material-ui/guides/typescript', title: 'TypeScript' },
-      { pathname: '/material-ui/guides/interoperability', title: 'Style library interoperability' },
-      { pathname: '/material-ui/guides/styled-components', title: 'Using styled-components' },
-      { pathname: '/material-ui/guides/theme-scoping' },
       { pathname: '/material-ui/guides/minimizing-bundle-size' },
-      { pathname: '/material-ui/guides/composition' },
       { pathname: '/material-ui/guides/routing' },
```

Comment:
To match the page title: https://mui.com/material-ui/guides/routing/.

```suggestion
      { pathname: '/material-ui/guides/routing', title: 'Routing libraries' },
```

Reply:
Also, this page is about how to integration with the link component of frameworks, so to move under integration?